evap/grades/templates/grades_course_view.html

```diff
@@ -29,7 +29,11 @@ <h3 class="mb-3">{{ course.name }} ({{ semester.name }})</h3>
                                 <td>
                                     <a href="{% url 'grades:download_grades' grade_document.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Download' %}"><span class="fas fa-download"></span></a>
                                     {% if user.is_grade_publisher %}
-                                        <a href="{% url 'grades:edit_grades' semester.id course.id grade_document.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Edit' %}"><span class="fas fa-pencil-alt"></span></a>
+                                        {% if "final" in grade_document.get_type_display %}
+                                            <a href="{% url 'grades:edit_grades' semester.id course.id grade_document.id %}?final=true" class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Edit' %}"><span class="fas fa-pencil-alt"></span></a>
```

Comment:
The view that we are linking to here will get the `grade_document.id` passed in and load the `GradeDocument` from the database anyway. Thus, we could just do all the logic in the view, and would not require the additional parameter -- or am I missing something?

Reply:
This change here is still part of the PR -- I guess you didn't add the file before committing?